static/app/views/organizationIntegrations/integrationRepos.tsx

```diff
@@ -84,7 +79,7 @@ export default class IntegrationRepos extends AsyncComponent<Props, State> {
   );
 
   searchRepositoriesRequest = (searchQuery: string) => {
-    const orgId = this.context.organization.slug;
+    const orgId = this.props.organization.slug;
```

Comment:
should we name the const 'orgSlug' ?

Reply:
Could, I wanted to focus on only removing legacy context right now though.